Whack-a-mole/main.js

```diff
@@ -0,0 +1,81 @@
+const holes = document.querySelectorAll(".hole");
+const scoreBoard = document.querySelector(".score");
+const moles = document.querySelectorAll(".mole");
+const level1 = document.querySelector(".easy");
+const level2 = document.querySelector(".medium");
+const level3 = document.querySelector(".hard");
+const record = document.querySelector(".record");
+let lastHole;
+let timeUp = false;
+let score = 0;
+let recording = localStorage.getItem("totalScore");
+
+
+window.onload = function (){
+    record.textContent = recording;
+    recording.textContent = localStorage.getItem("totalScore");
```

Comment:
what does this code do?

Reply:
This code loading value localstorage when  load page. 
I'm update this code 
window.onload = function (){
    record.textContent = localStorage.getItem("totalScore");
 };

Reply:
Does `recording` have `textContent`?